spec/function.dd

```diff
@@ -670,11 +670,15 @@ $(H2 $(LNAME2 inout-functions, Inout Functions))
         ---
         )
 
-    $(P The $(D_KEYWORD inout) token forms a wildcard that stands in for
-        any of mutable, `const`, `immutable`, `inout`, or `inout const`. When calling
-        the function, the `inout` of the return type is changed to whatever
-        the mutable, `const`, `immutable`, `inout`, or `inout const` status of the
-        argument type to the `inout` parameter was.
+    $(P The $(D_KEYWORD inout) keyword forms a wildcard that stands in for
+        one of these modifiers: mutable, `const`, `immutable`, `inout`, or `inout const`.
```

Comment:
There is no mutable modifier, so I don't think it's accurate to put in a list of "one of these modifiers". I think it's fine to drop "these modifiers" and just leave it at that. Or, alternatively, use "states" instead.